src/components/__tests__/FlagBox.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import FlagBox from '../../components/FlagBox'
+
+const App = () => {
```

Comment:
Would you mind telling me a bit about those tests? At a glance, looks like they are mainly accessories to make sure that `tsc` is satisfied with the type definitions, is that correct?

Reply:
@mcataford ✔️ Correct! It's to ensure that the type definitions match the expected use cases of the components.